app/authentication_views.py

```diff
@@ -5,63 +5,95 @@
 from app import register_validation as validation
 from app.data.repositories import UsersRepository
 
+from app.logs.logger import logger
+
 
 # TODO add  generic error messages
 @app.route("/register", methods=["POST", "GET"])
 def register():
 
+    logger.info("Registering, route /register is called")
     if "user" in session:
+        logger.info("Registering, user is authenticated")
+        logger.info("Registering, redirecting to route 'home'")
         return redirect(url_for("home"))
 
     elif request.method == "POST":
+        logger.info("Registering, register form is submitted")
 
         username = request.form['username']
         email = request.form['email']
         password = request.form['password']
         birthday = request.form['birthday']
 
+        logger.info(
+            f"Registering, username: {username}, email: {email}, password: {password}, birthday: {birthday}")
+
         if not validation.validate_date_format(birthday):
+            logger.error(
+                f"Registering, registering new user {username} failed: invalid date format")
             return "wrong date format"
 
         if UsersRepository.get_by_username_email(username, email):
+            logger.error(
+                f"Registering, registering new user {username} failed: User already exits")
             return "User already exits"
 
         if not validation.validate_password(password):
+            logger.error(
+                f"Registering, registering new user {username} failed: invalid password")
             return "at least 1 capital letter, 1 small letter, 1 number, length 8 - 20"
 
         if not validation.validate_email(email):
+            logger.error(
+                f"Registering, registering new user {username} failed: invalid email format")
             return "wrong email format"
 
         if UsersRepository.create(username, email, crypto.hash_password(password), birthday):
+            logger.info(
+                f"Registering, registering new user {username} is finished")
             return "success"
 
+        logger.error(f"Registering, registering new user {username} failed")
         return "Something went wrong"
 
     else:
+        logger.info("Registering, rendering register.html")
         return render_template("forms/register.html")
 
 
 @app.route("/login", methods=["POST", "GET"])
 def login():
 
+    logger.info("Loging, route '/login' is called")
```

Comment:
Logging and Logging word alone doesnt mean much
change context to: 'Handling /login route'?

Reply:
done